__tests__/app.test.js

```diff
@@ -45,4 +53,29 @@ describe("GET Requests", () => {
         });
     });
   });
+
+  describe("/api/articles/:article_id", () => {
+    test("status 200: responds with an article object with given id and all properties", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then(({ body }) => {
+          const { article } = body;
+
+          expect(article).toBeInstanceOf(Object);
+
+          expect(article).toEqual(
+            expect.objectContaining({
+              author: expect.any(String),
+              title: expect.any(String),
+              article_id: expect.any(Number),
```

Comment:
You want to check that you actually got the article with article_id of `1` here so you can hardcode it instead of saying expect.any(Number)!